### DISCUSSION TITLE
How do I fix resource <URL> Link Preload Warning in my Gatsby Project?

### DISCUSSION THREAD

**Discussion by Pochetes** [gatsbyjs/gatsby#35566] [Help]
Whenever I run the server for my Gatsby application, I receive these lighthouse warnings:

![Screen Shot 2022-05-02 at 12 21 30 AM](https://user-images.githubusercontent.com/63836051/166183944-f26394c5-0529-4c18-9083-0ea455c8892f.png)

Some of the resources are:

`http://localhost:8000/socket.io/socket.io.js`

`http://localhost:8000/commons.js`

`http://localhost:8000/page-data/app-data.json`

What could this error be preluding to? I've read many issue threads saying that it could be over-prioritization of resource loading which increases bandwidth. Others say it's related to font preloading (I use Tailwind CSS if that helps). Should I progressively load my components as one scrolls down the page?

My Specs
  System:
    OS: macOS 12.3.1
    CPU: (8) arm64 Apple M1
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 17.3.1 - /opt/homebrew/bin/node
    Yarn: 1.22.10 - /opt/homebrew/bin/yarn
    npm: 8.5.4 - /opt/homebrew/bin/npm
  Languages:
    Python: 3.9.9 - /opt/homebrew/opt/python@3.9/libexec/bin/python
  Browsers:
    Chrome: 100.0.4896.127
    Safari: 15.4
  npmPackages:
    gatsby: ^4.4.0 => 4.7.2
    gatsby-plugin-image: ^2.13.0 => 2.13.0
    gatsby-plugin-manifest: ^4.0.0 => 4.7.0
    gatsby-plugin-mdx: ^3.5.2 => 3.7.1
    gatsby-plugin-postcss: ^5.6.0 => 5.7.0
    gatsby-plugin-sharp: ^4.13.0 => 4.13.0
    gatsby-plugin-webpack-bundle-analyser-v2: ^1.1.22 => 1.1.26
    gatsby-source-filesystem: ^4.13.0 => 4.13.0
    gatsby-transformer-sharp: ^4.13.0 => 4.13.0
  npmGlobalPackages:
    gatsby-cli: 4.6.

**Comment by LekoArts** [comment]
You'll want to run lighthouse tests on `gatsby serve` after `gatsby build` (if you want to test it locally). But that's also not a good measurement, you'll want to test it deployed on a CDN.

Lighthouse tests on `gatsby develop` won't give you any real-world data so you can skip that.
